install_config/storage_examples/ceph_example.adoc

```diff
@@ -26,79 +26,93 @@ storage.
 
 [NOTE]
 ====
-All `oc ...` commands are executed on the {product-title} master host.
+All `oc` commands are executed on the {product-title} master host.
 ====
 
-[[using-ceph-rbd-installing-the-ceph-common-package]]
-== Installing the ceph-common Package
-The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
-
-[NOTE]
-====
-The {product-title} all-in-one host is not often used to run pod workloads and,
+[discrete]
+== Prerequisites
+* Install the latest ceph-common package using `yum install -y ceph-common`. The
+*ceph-common* library must be installed on *all schedulable* {product-title}
+nodes.
+* The {product-title} all-in-one host is not often used to run pod workloads and,
 thus, is not included as a schedulable node.
-====
 
+[[using-existing-ceph-cluster-as-persistent-store]]
+== Using an existing Ceph cluster as a persistent store
+
+To use an existing Ceph cluster as a persistent store:
+
+.  Create the keyring for the client:
++
+[source, bash]
 ----
-# yum install -y ceph-common
+$ ceph auth get-or-create client.openshift-test mon 'allow *' osd 'allow rwx pool=openshift' -o /etc/ceph/ceph.client.openshift-test-keyring
 ----
 
-[[using-ceph-rbd-creating-the-ceph-secret]]
-== Creating the Ceph Secret
-
-The `ceph auth get-key` command is run on a Ceph *MON* node to display the key
-value for the *client.admin* user:
+. Convert the keyring to base64:
++
+[source, bash]
+----
+$ /etc/ceph/ceph.client.openshift-test-keyring
+[client.openshift-test]
+	key = AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ==
+$ echo -n AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ== |base64
+QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQ==
+----
 
-.Ceph Secret Definition
-====
+.  Create the ceph-secret.yaml with the base64-decoded keyring:
++
+[source, bash]
+----
+$ ceph-secret.yaml
+----
++
+.Ceph secret definition example
 [source,yaml]
 ----
 apiVersion: v1
 kind: Secret
 metadata:
-  name: ceph-secret
+  name: ceph-secret-test
 data:
-  key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
-
+  key: QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQo= <1>
 ----
-<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth get-key client.admin | base64` command, then copying the output and pasting it as the secret key's value.
-====
-
-Save the secret definition to a file, for example *_ceph-secret.yaml_*,
-then create the secret:
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph
```

Comment:
I'd update this note in the same way as the other topic.
And I still think that the yaml goes before the command to create the object. (all instances)